Source/Fuse.Animations/Attract.uno

```diff
@@ -14,7 +14,6 @@ namespace Fuse.Animations
 		A single `AttractorConfig` can be used for multiple `attract` expressions.
 		
 		@see @Attract
-		@experimental
```

Comment:
This one isn't in Fuse.Charting, did you mean to include it?

Reply:
`AttractorConfig` is used a lot in Fuse.Charting code so I think we should drop `@experimental` here too.

Reply:
Ok. Perhaps you could include it in the commit message?

Reply:
Fixed.